controllers/Backend.php

```diff
@@ -1,104 +1,120 @@
+        $myView = new View('dashboard', 'backend');
+        $myView->render(array('comments' => $comments));
     }
 
     public function write()
     {
-        $model_article = new Article();
-        $model_user = new User();
-        
-        $page="backend/write";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        $manager_article = new ArticleManager();
+        $errors = $manager_article->write_verification();
+    
+        $myView = new View('write', 'backend');
+        $myView->render(array('errors' => $errors));
     }
 
     public function settings()
     {
-        $model_user = new User();
-        
-        $page="backend/settings";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        $manager_user = new UserManager();
+        $modos = $manager_user->get_modos();
+        $errors = $manager_user->settings_verification();
+ 
+        $myView = new View('settings', 'backend');
+        $myView->render(array('modos' => $modos, 'errors' => $errors));
     }
 
     public function list()
     {
-        $model_article = new Article();
-        $model_user = new User();
-
-        $page="backend/list";
-        $topbar="backend/topbar";
-        require "views/layout.php";
-    }
-
-    public function logout()
-    {
-        $model_user = new User();
+        $manager_article = new ArticleManager();
+        $posts = $manager_article->get_posts();
+        
+        $manager_user = new UserManager();
+        $users = $manager_user->admin();
+        
+        if($users=0){
+        header("Location:/dashboard");
+        }
         
-        $page="backend/logout";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        $myView = new View('list', 'backend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function password()
     {
-        $model_user = new User();
-        
-        $page="backend/password";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        $manager_user = new UserManager();
+        $errors = $manager_user->password_verification();
+   
+        if($manager_user->hasnt_password() == 0){
+        header("Location:/dashboard");
+    }
+        $myView = new View('password', 'backend');
+        $myView->render(array('errors' => $errors));
     }
 
     public function postback()
     {
-        $model_article = new Article();
-        $model_user = new User();
-        
-        $page="backend/postback";
-        $topbar="backend/topbar";
-        require "views/layout.php";
+        if(isset($_GET['id']))
+        {
+        $id = $_GET['id'];
+        $manager_article = new ArticleManager();
+        $posts = $manager_article->get_post();
+        $errors = $manager_article->postback_verification();
+        }
+
+        if($posts == false){
+            header("Location:/page-erreur-administrateur");
+        }
+
+        else{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method postback uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)